pkg/genericapiserver/server/options/server_run_options.go

```diff
@@ -66,7 +66,7 @@ type ServerRunOptions struct {
 func NewServerRunOptions() *ServerRunOptions {
 	return &ServerRunOptions{
 		AdmissionControl:            "AlwaysAdmit",
-		DefaultStorageMediaType:     "application/json",
```

Comment:
Agree. If all kube apiservers use protobufs, put the kube default into pkg/kubeapiserver and reference that from the call sites of `genericoptions.NewServerRunOptions` (e.g. https://github.com/kubernetes/kubernetes/blob/b511741df76a7bf31683361999915c5e74eefcbe/cmd/kube-apiserver/app/options/options.go#L90-L90). A comment above/behind `DefaultStorageMediaType` here about how to enable protobufs wouldn't harm.